2022q4/pantheon.adoc

```diff
@@ -22,20 +23,21 @@ Since the previous report, we have been updating its related ports, especially:
 * `multimedia/elementary-videos`
 * `x11/elementary-terminal`
 * `x11-themes/gnome-icons-elementary`
-* `x11-toolkits/granite7`
+* `x11-toolkits/granite7`.
 
 Power manager plugins for top panel (wingpanel) and control center (switchboard) are finished.
 
-A new switchboard plugin is also available, `net/switchboard-plug-sharing`. Ported *Rygel*, GNOME UPnP/DLNA services.
+A new switchboard plugin is also available, `net/switchboard-plug-sharing`.
+Ported *Rygel*, GNOME UPnP/DLNA services.
 
 Submitted other patches for low level libraries such as:
 
 * `print/cups-pk-helper` update to link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266067[0.2.7] required by `print/switchboard-plus-printers`
 * `devel/libgee` update to link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266585[0.20.6] heavily uses by the desktop
 * `sysutils/polkit` update to 122 link:https://reviews.freebsd.org/D37137[D37137]
-* `sysutils/accountsservice` update to 22.08.8 link:https://reviews.freebsd.org/D37679[D37679]
+* `sysutils/accountsservice` update to 22.08.8 link:https://reviews.freebsd.org/D37679[D37679].
 
 ==== Open tasks
 
 * Improve documentation
-* Continue to work on user settings
+* Continue to work on user settings.
```

Comment:
Consistent with another comment I made above, I would either drop the period on line 43 or add one to each list element.